src/components/reports/FieldSet.tsx

```diff
@@ -30,11 +38,11 @@ const FieldSet: FC<Props> = ({ name, label, selectOptions, ...rest }) => {
   }
 
   return (
-    <>
+    <span>
       <label htmlFor={name}>{label}</label>
       {field}
       <ErrorMessage name={name} component="p" className="error-text" />
```

Comment:
Here, I was thinking more that the `span` could be passed to the `component` prop like this:

```suggestion
    <>
      <label htmlFor={name}>{label}</label>
      {field}
      <ErrorMessage name={name} component="span" className="error-text" />
    </>
```

Reply:
aha right! That makes more sense